### PR TITLE
Spring cleaning

### PR DESCRIPTION
**Pull request by stoeffel**

### CODE REVIEW

File: brangelina.vim [hunk 1]
```diff
@@ -229,22 +188,18 @@ function! BrangelinaPlugins()
   Plug 'junegunn/vader.vim'                  "  vim test framework
   Plug 'justinhoward/fzf-neoyank'
   Plug 'justinmk/vim-sneak'                  "  Medium-range motion
```
Comment (stoeffel) [inline brangelina.vim]: @jwoudenberg do you ever use this?

File: brangelina.vim [hunk 2]
```diff
@@ -168,51 +149,29 @@ augroup customCommands
...
 " # Plugins
 function! BrangelinaPlugins()
   Plug 'Shougo/deoplete.nvim'
   Plug 'Shougo/neoyank.vim'
   Plug 'airblade/vim-gitgutter'              "  Column with line changes
   Plug 'amiorin/vim-fenced-code-blocks'      "  Edit code in Markdown code blocks
+  Plug 'arithran/vim-delete-hidden-buffers'
   Plug 'bronson/vim-visual-star-search'      "  Easily search for the selected text
   Plug 'editorconfig/editorconfig-vim'       "  Settings based on .editorconfig file
   Plug 'elentok/todo.vim'                    "  Todo.txt support
```
Comment (stoeffel) [inline brangelina.vim:149]: Do you use this?

File: brangelina.vim [hunk 3]
```diff
@@ -168,51 +149,29 @@ augroup customCommands
...
 command! -bang -nargs=? -complete=dir Files
-  \ call fzf#vim#files(<q-args>, fzf#vim#with_preview({ 'options': '--bind ctrl-a:select-all,ctrl-d:deselect-all,ctrl-q:toggle-preview' }, 'up:60%'), <bang>0)
+  \ call fzf#vim#files(<q-args>, { 'options': '--bind ctrl-a:select-all,ctrl-d:deselect-all' }, <bang>0)
 
 " # Plugins
 function! BrangelinaPlugins()
   Plug 'Shougo/deoplete.nvim'
   Plug 'Shougo/neoyank.vim'
   Plug 'airblade/vim-gitgutter'              "  Column with line changes
   Plug 'amiorin/vim-fenced-code-blocks'      "  Edit code in Markdown code blocks
```
Comment (stoeffel) [inline brangelina.vim]: do we need this?

File: brangelina.vim [hunk 4]
```diff
@@ -168,51 +149,29 @@ augroup customCommands
...
+  \   { 'options': '--bind ctrl-a:select-all,ctrl-d:deselect-all' },
   \   <bang>0)
 
 command! -bang -nargs=? -complete=dir Files
-  \ call fzf#vim#files(<q-args>, fzf#vim#with_preview({ 'options': '--bind ctrl-a:select-all,ctrl-d:deselect-all,ctrl-q:toggle-preview' }, 'up:60%'), <bang>0)
+  \ call fzf#vim#files(<q-args>, { 'options': '--bind ctrl-a:select-all,ctrl-d:deselect-all' }, <bang>0)
 
 " # Plugins
 function! BrangelinaPlugins()
   Plug 'Shougo/deoplete.nvim'
```
Comment (stoeffel) [inline brangelina.vim]: deoplete breaks every month or so for me would love to have a better plugin

**Comment by stoeffel** [comment]
Isn't this handled by the editorconfig plugin? 
https://github.com/stoeffelberg/brangelina/blob/899c945c40427d655302765e387286eb9870f3bc/brangelina.vim#L135-L136

**Comment by stoeffel** [comment]
I never remember any of the localleader mappings. I wonder if we should drop those:

https://github.com/stoeffelberg/brangelina/blob/899c945c40427d655302765e387286eb9870f3bc/brangelina.vim#L138-L151

**Comment by stoeffel** [comment]
can we use https://github.com/tpope/vim-sensible/blob/master/plugin/sensible.vim and remove some `set`? #58

File: brangelina.vim (hunk 1 above)
Comment (jwoudenberg) [reply brangelina.vim]: Nope! Let's get rid of it.

**Comment by stoeffel** [comment]
@jwoudenberg sorry, this will take some time to review. ❤️

File: brangelina.vim (hunk 2 above)
Comment (jwoudenberg) [reply brangelina.vim:149]: Yes, this one I use!

File: brangelina.vim (hunk 3 above)
Comment (jwoudenberg) [reply brangelina.vim]: Forgot we had this. I think we can remove it.

File: brangelina.vim (hunk 4 above)
Comment (jwoudenberg) [reply brangelina.vim]: Yeah. I'm also often confronted with a string of error messages everytime I close vim.

**Comment by jwoudenberg** [comment]
> Isn't this handled by the editorconfig plugin?

Only if we have an `.editorconfig` file present. I usually don't but would still like to strip trailing whitespace by default.

**Comment by jwoudenberg** [comment]
Looks good to me! I played around with it a little and don't notice any problems. Happy to merge this!

Thanks for all your work in these PRs stoeffel, love the new brangelina!

**Comment by stoeffel** [comment]
btw. I just have a global editorconfig to remove trailing spaces

**Comment by stoeffel** [comment]
merging this and I will write some docs as well soon

**Pull request merged**